cmd/restic/secondary_repo_test.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	resticTest "github.com/restic/restic/internal/test"
+)
+
+//secondaryRepoTestCase defines a struct for test cases of secondary repositories
+type secondaryRepoTestCase struct {
+	Opts     secondaryRepoOptions
+	GOpts    GlobalOptions
+	DstGOpts GlobalOptions
+}
+
+//validSecondaryRepoTestCases is a list with test cases that must pass
+var validSecondaryRepoTestCases = []secondaryRepoTestCase{
+	{
+		// Test if Repo and Password are parsed correctly.
+		Opts: secondaryRepoOptions{
+			Repo:     "backupDst",
+			password: "secretDst",
+		},
+		GOpts: GlobalOptions{
```

Comment:
`GOpts` seems to be identical for all tests cases. Maybe define these GlobalOptions once and the use them for each test case?